### DISCUSSION TITLE
Featured images on blog posts/articles.

### DISCUSSION THREAD

**Discussion by axjms1** [willfaught/paige#32] [Ideas]
Implementing something like this could be nice.  I am working on it on my paige site. 

https://makewithhugo.com/thumbnail-images-on-your-hugo-blog-posts/

**Comment by willfaught** [comment]
Please see the densities, breakpoints, process, and linked parameters in the paige/gallery and paige/image shortcodes. Is there something you're trying to accomplish that those don't fit?

**Reply by axjms1** [reply]
Thank you, Will.  I have looked closely over those and I could make a page consisting of a gallery of images that are links to each article.  But the images aren't that important and I would lose the convenience of the title and description being listed for each post.  I am simply trying to add visual interest.  For example, in your blog here [https://willfaught.com/blog/](https://willfaught.com/blog/) it would be nice to have a small featured image to the left of each post if you so choose.

**Reply by willfaught** [reply]
Oh, I see what you mean. No, Paige doesn't show thumbnails in page lists. How would you want it to be displayed in terms of rows and cols per page? How would wrapping work for long page titles and descriptions? It wasn't obvious how to answer these in a way I liked, so I kept it simple.
